src/cfg.cpp

```diff
@@ -347,6 +349,10 @@ void read_cfg()
 	if (external_ambient_ext.empty() || external_ambient_ext.front().empty())
 		external_ambient_ext = std::vector<std::string>(1, "ogg");
 
+	// EXTERNAL IBL PATH
+	if (external_ibl_path.empty())
+		external_ibl_path = "lighting/IBL";
```

Comment:
For the sake of convention, can you please make the path all lowercase? Thank you :)

Reply:
Sorry, my mistake, I changed it to lowercase.